sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -75,10 +75,23 @@ static void usb_receipt_callback_1(uint8_t c) {
   ++usb_chars_recved_total;
 }
 
+static void usb_send_str(char *s, usb_ss_ctx_t *ss) {
+  while (*s) usb_simpleserial_send_byte(ss, *s++);
```

Comment:
Could you rewrite this as:
```
for (int i = 0; s[i] != 0; ++i) {
  usb_simpleserial_send_byte(ss, s[i]);
}
```
This is more consistent with similar code under SW.

Reply:
Done